lib/nerves_hub/deployments/calculator.ex

```diff
@@ -121,9 +122,10 @@ defmodule NervesHub.Deployments.Calculator do
 
     Logger.info("[InflightDeploymentCheck] reloaded deployment")
 
-    if state.process_timer_ref do
-      Process.cancel_timer(state.process_timer_ref)
-    end
+    _ =
+      if state.process_timer_ref do
+        Process.cancel_timer(state.process_timer_ref)
+      end
```

Comment:
This is how I've been fixing this lately. Ignoring `if`'s always looks weird to me.

```suggestion 
    if state.process_timer_ref do
      _ = Process.cancel_timer(state.process_timer_ref)
      :ok
    end
```